tests/functional/z_cluster/test_multiple_mds.py

```
@@ -178,3 +186,226 @@ def test_node_drain_and_fault_tolerance_for_multiple_mds(self, pod_factory):
+        """
+        Verify MDS failover does not break CephFS IO on an EC data pool.
+
+        Steps:
+        1. Scale MDS from 1 to 2 (so a standby-replay is available)
+        2. Create CephFS RWX PVC on EC pool and start long-running IO
+        3. Fail the active MDS daemon via 'ceph mds fail'
+        4. Verify standby takes over (active + standby counts unchanged)
+        5. Verify IO completes without errors
+        """
+        with config.RunWithProviderConfigContextIfAvailable():
+            sc_obj, full_pool_name = cephfs_ec_storageclass
```

Ruff (0.15.21)

[warning] Unpacked variable `full_pool_name` is never used

Prefix it with an underscore or any other dummy variable pattern

(RUF059)